components/FormContainer.vue

```diff
@@ -0,0 +1,271 @@
+			return [
+				{
+					component: FormIntro,
+					introduction: 'Hi! 👋 This is an invitation to connect to other MozFest participants via skills, interests and fascinations.',
+					followup: 'Your input will be used to connect you with other participants on Slack and will be shown on the MozFest Gradual page.',
+					updateFunction: () => { }
+				},
+				{
+					component: FormSingleField,
+					introduction: "What's your @UserName on the MozFest Slack?",
+					followup: "Please write down your exact user name, including the @ sign.",
+					value: this.formData.username,
+					updateFunction: (value) => {
+						this.setSingleField('username', value)
+					}
+				},
+				{
+					component: FormMultipleSelect,
+					introduction: "These are some topics suggested by other Mozfest participants! Anything here you'd like to learn more about?",
+					options: this.currentLearning,
+					value: this.formData.learnings,
+					updateFunction: (learnings) => {
+						this.setSingleField('learnings', learnings)
+					}
+				},
+				{
+					component: FormMultipleSelect,
+					introduction: "Here are some popular topics MozFest participants would like to learn. Anything here you could share something about?",
+					value: this.formData.sharings,
+					options: this.currentSharing,
+					updateFunction: (sharings) => {
+						this.setSingleField('sharings', sharings)
+					}
+				},
+				{
+					component: FormMultipleInput,
+					introduction: "Are there any other things you would like to learn?",
+					followup: "Please write it in a way others will understand. For example: 'how to start a book club'.",
+					value: this.formData.newLearnings,
+					updateFunction: (value) => {
+						this.setSingleField('newLearnings', value)
+					}
+				},
+				{
+					component: FormMultipleInput,
+					introduction: "Now, are there skills you would like to share?",
+					followup: "Think stories, skills, methods, shortcuts, failures, successes or tips and tricks. Anything goes, don't be shy!",
+					value: this.formData.newSharings,
+					updateFunction: (value) => {
+						this.setSingleField('newSharings', value)
+					}
+				},
+				{
+					component: FormSingleField,
+					introduction: `Thank you, ${this.formData.username}!`,
+					followup: "	Anything else you want to tell us before you go?",
+					value: this.formData.comment,
+					updateFunction: (value) => {
+						this.setSingleField('comment', value)
+					},
+					textarea: true
+				},
+			]
+		},
+		hasNextState() {
+			return this.stateIndex < this.states.length - 1
+		},
+		hasPreviousState() {
+			return this.stateIndex > 0
+		},
+		currentState() {
+			return this.states[this.stateIndex]
+		},
+		percentage() {
+			return Math.ceil(((this.stateIndex + 1) / (this.states.length)) * 100)
+		}
+	},
+	components: {
+		FormIntro,
+		FormSingleField,
+		FormMultipleInput,
+		FormMultipleSelect,
+		ProgressBar
+	},
+	methods: {
+		nextStep() {
+			if (this.hasNextState) {
+				this.stateIndex += 1
```

Comment:
In the case of FormMultiSelect, scrolling down to the bottom and pressing the "Next" button, progresses to the next state, but keeps the scrollheight. This is confusing because the state's introduction is not visible. 
Could we trigger a scroll back to top as part of nextStep() ?

Reply:
Done.